WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostDetailFragment.kt

```diff
@@ -997,11 +980,12 @@ class ReaderPostDetailFragment : ViewPagerFragment(),
                 }
             }
         } else {
-            reblogButton.visibility = View.GONE
+            reblogButton.isEnabled = false
             reblogButton.setOnClickListener(null)
         }
 
         if (canShowCommentCount()) {
+            countLikes.isEnabled = true
```

Comment:
Could you please double check this line: Should this be "countComments"?

Reply:
Nice catch, thank you! Fixed in 7bd6f38